microsoft-office-365-feed/info.json

```diff
@@ -0,0 +1,124 @@
+{
+  "name": "microsoft-office-365-feed",
+  "label": "Microsoft Office 365 Feed",
+  "version": "1.0.0",
+  "description": "Ingest the IP, URLs used by Office 365 using the web service provided by Microsoft. The fetched indicators can be used to create a whitelist, blacklist etc. for your SIEM or firewall services.",
```

Comment:
\nRequires the 'Threat Intel Management Solution Pack' to be pre-installed for the data ingestion capabilities to work correctly